bears/general/IndentationBear.py

```diff
@@ -386,6 +399,23 @@ def get_valid_sequences(file, sequence, annotation_dict):
                    lt_eq(sequence_position, comment.end)):
                     valid = False
 
+                if check_ending:
+                    if(comment.start.line == sequence_position.line and
+                            comment.end.line == sequence_position.line):
+                        sequence_line_text = sequence_line_text[
+                            :comment.start.column - 1] + sequence_line_text[
+                            comment.end.column-1:]
+
+            if encapsulators:
+                for encapsulator in encapsulators:
+                    if(gt_eq(sequence_position, encapsulator.start) and
```

Comment:
Could be a new commit. You can freely alter commits of your PR though, it's 
not a Problem for us.


Am 23. Juli 2016 8:43:18 vorm. schrieb Abhay Raizada 
<notifications@github.com>:

>> @@ -386,6 +399,23 @@ def get_valid_sequences(file, sequence, annotation_dict):
>>                     lt_eq(sequence_position, comment.end)):
>>                      valid = False
>>
>> +                if check_ending:
>> +                    if(comment.start.line == sequence_position.line and
>> +                            comment.end.line == sequence_position.line):
>> +                        sequence_line_text = sequence_line_text[
>> +                            :comment.start.column - 1] + sequence_line_text[
>> +                            comment.end.column-1:]
>> +
>> +            if encapsulators:
>> +                for encapsulator in encapsulators:
>> +                    if(gt_eq(sequence_position, encapsulator.start) and
>
> that should be in a new commit right? since it also deals with earlier 
> lines of code?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/coala-analyzer/coala-bears/pull/570/files/546df9f6ff0e550b1c6ba972cdff2091b9f83a8c..bd456e2b4e30f42326cf0eab14916b7e1e92d0aa#r71968448